ci-operator/step-registry/fips-check/optional-operator-images-scan/fips-check-optional-operator-images-scan-commands.sh

```diff
@@ -130,17 +130,23 @@ for package in ${package_list}; do
         csv_test_result="${data_dir}/${package}/$csv"
         echo "##Starting scan for CSV '$csv'" >> $csv_test_result
         # retrieve related images from current CSV
-        image_list=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' || true);
+        image_list_string=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' | sort | uniq || true);
+        image_list_string=$(echo $image_list_string)
+        read -r -a image_list <<< ${image_list_string}
         if $internal; then
-            image_list=$(echo $image_list | sed -r 's/registry.redhat.io/brew.registry.redhat.io/g' )
-            image_list=$(echo $image_list | sed -r 's/registry.stage.redhat.io/brew.registry.stage.redhat.io/g')
+            len=${#image_list[@]}
+            for ((i=0;i<len;i++)); do
+                if [[ ${image_list[$i]} != *"brew"* ]]; then
+                    image_list[$i]=$(echo ${image_list[$i]} | sed -r 's/registry.redhat.io/brew.registry.redhat.io/g' )
+                    image_list[$i]=$(echo ${image_list[$i]} | sed -r 's/registry.stage.redhat.io/brew.registry.redhat.io/g')
+                 fi
+            done
         fi
```

Comment:
More concise way:
```
        image_list_string=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' | sort | uniq || true)
        if $internal; then
            # [1]
            image_list_string=$(echo "$image_list_string" | sed 's/^registry.redhat.io/brew.registry.redhat.io/g' )
            image_list_string=$(echo "$image_list_string" | sed 's/^registry.stage.redhat.io/brew.registry.stage.redhat.io/g')
        fi
        # [2]
        readarray -t image_list <<< "${image_list_string}"
```
With [1] , need not the `for` loop.
With [2],  ` image_list_string=$(echo $image_list_string)` can be removed. `readarray` is a built-in shell command like `read`, see `man readarray`.